app/screens/channel_add_members/channel_add_members.js

```diff
@@ -264,15 +241,22 @@ class ChannelAddMembers extends PureComponent {
                     style={{marginVertical: 5}}
                 >
                     <SearchBar
-                        ref={this.searchBarRef}
-                        placeholder={formatMessage({id: 'search_bar.search', defaultMesage: 'Search'})}
-                        height={27}
-                        fontSize={14}
-                        textColor={changeOpacity('#000', 0.5)}
-                        hideBackground={true}
-                        textFieldBackgroundColor={'#fff'}
-                        onChange={this.searchProfiles}
-                        onSearchButtonPress={this.onSearchButtonPress}
+                        ref='search_bar'
+                        placeholder={formatMessage({id: 'search_bar.search', defaultMessage: 'Search'})}
+                        cancelTitle={formatMessage({id: 'mobile.post.cancel', defaultMessage: 'Cancel'})}
+                        backgroundColor='transparent'
+                        inputHeight={30}
+                        inputStyle={{
```

Comment:
oh now that I see the code, this aren't styles.. this are properties and this values are here while @asaadmahmood decides what are going to be the colors to use in the search bar, so i'm leaving this as is